packages/instrumentation-typeorm/package.json

```diff
@@ -23,7 +23,7 @@
     "scripts": {
         "build": "tsc",
         "prepare": "yarn run build",
-        "test": "jest",
+        "test": "mocha --require ts-node/register",
```

Comment:
the --required flag is not found in other instrumentation libraries.
Is it something we need specifically in this one?

Reply:
thanks. fixed.